gravatar-quickeditor/src/main/res/values/available_languages.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--Warning: Auto-generated file, do not edit.-->
 <resources xmlns:tools="http://schemas.android.com/tools">
-  <string-array name="available_languages" translatable="false" tools:ignore="InconsistentArrays">
+  <string-array name="available_languages" translatable="false" tools:ignore="InconsistentArrays, UnusedResources">
```

Comment:
Is this ok @iangmaia ? I don't know how GlotPress integration works: is this file automatically generated, or is it fine to add configuration to it like this?

Reply:
It was originally auto-generated, but I think this will be fine (as in: I don't think this will be overwritten, only the locale list should be).